### DISCUSSION TITLE
关于索引号和文字间距的问题

### DISCUSSION THREAD

**Discussion by YanJun1115** [sjtug/SJTUThesis#914] [提问] [answered]
如题，索引号到两位数以后，数字和文字的间距特别小，请问这要怎么修改？
![1](https://user-images.githubusercontent.com/61675058/230719960-4b9e84ea-01b8-4bfe-a2df-b47c7f1b9494.jpg)

**Comment by LogCreative** [comment, marked as answer]
该问题已在 2.0.2 版本中修复，请使用 GitHub 上下载的最新模板。

**Reply by YanJun1115** [reply]
好的，非常感谢！

**Reply by iquitap** [reply]
想请教一下具体是哪个哪个commit的修改对应这个内容呢？

**Reply by LogCreative** [reply]
87e18875473a175a20719a5e97d0f6a22002b410

**Reply by iquitap** [reply]
非常感谢您及时的答复👌
